src/configuration/OpenSearchDetails.test.tsx

```diff
@@ -1,21 +1,45 @@
 import React from 'react';
 import { mount } from 'enzyme';
 import { OpenSearchDetails } from './OpenSearchDetails';
-import { createDefaultConfigOptions } from './mocks';
+import { createDefaultConfigOptions } from '__mocks__/DefaultConfigOptions';
 import { LegacyForms } from '@grafana/ui';
 import { Flavor, OpenSearchOptions } from 'types';
 import { last } from 'lodash';
 import { DataSourceSettings } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { setupMockedDataSource } from '__mocks__/OpenSearchDatasource';
 const { Select, Switch } = LegacyForms;
 
+const opensearchDetectVersionValue = config.featureToggles.opensearchDetectVersion;
+
+afterAll(() => {
```

Comment:
does this work when it's outside of the describe? it might have to be inside? Please go ahead and tell me if I'm wrong tho haha